src/components/NFTViewOptionList.vue

```diff
@@ -112,11 +112,19 @@ export default {
     },
   },
   computed: {
+    normalizedContentURLs() {
+      return this.contentUrls.length ? this.contentUrls : [this.url];
```

Comment:
```suggestion
      // NOTE: Assuming if only `url` is set, it must contain the actual content rather than the book info
      return this.contentUrls.length ? this.contentUrls : [this.url];
```